updating/updating-restricted-network-cluster.adoc

```diff
@@ -31,43 +33,147 @@ If you are running cluster monitoring with an attached PVC for Prometheus, you m
+=== Preparing your mirror host
 
 Before you perform the mirror procedure, you must prepare the host to retrieve content
 and push it to the remote location.
 
-include::modules/cli-installing-cli.adoc[leveloffset=+2]
+include::modules/cli-installing-cli.adoc[leveloffset=+3]
 
 // this file doesn't exist, so I'm including the one that should pick up more changes from Clayton's PR - modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
 
-include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+1]
+include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
 
-include::modules/update-mirror-repository.adoc[leveloffset=+1]
+include::modules/update-mirror-repository.adoc[leveloffset=+2]
 
 [id="updating-restricted-network-image-signature-configmap"]
-== Creating the image signature config map
+=== Creating the image signature config map
 
 Before you update your cluster, you must manually create a config map that contains the signatures of the release images that you use. This signature allows the Cluster Version Operator (CVO) to verify that the release images have not been modified by comparing the expected and actual image signatures.
 
 If you are upgrading from version 4.4.8 or later, you can use the `oc` CLI to create the config map. If you are upgrading from an earlier version, you must use the manual method.
 
-include::modules/update-oc-configmap-signature-verification.adoc[leveloffset=+2]
+include::modules/update-oc-configmap-signature-verification.adoc[leveloffset=+3]
 
-include::modules/update-configuring-image-signature.adoc[leveloffset=+2]
+include::modules/update-configuring-image-signature.adoc[leveloffset=+3]
 
-include::modules/machine-health-checks-pausing.adoc[leveloffset=+1]
+include::modules/machine-health-checks-pausing.adoc[leveloffset=+3]
 
-include::modules/update-restricted.adoc[leveloffset=+1]
+include::modules/update-restricted.adoc[leveloffset=+2]
 
-include::modules/images-configuration-registry-mirror.adoc[leveloffset=+1]
+include::modules/images-configuration-registry-mirror.adoc[leveloffset=+2]
 
-include::modules/generating-icsp-object-scoped-to-a-registry.adoc[leveloffset=+1]
+include::modules/generating-icsp-object-scoped-to-a-registry.adoc[leveloffset=+2]
 
 [id="additional-resources_security-container-signature"]
-== Additional resources
+=== Additional resources
 
 * xref:../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks]
 
 * xref:../post_installation_configuration/machine-configuration-tasks.adoc#machine-config-overview-post-install-machine-configuration-tasks[Machine Config Overview]
 
 * xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]
+
+== Updating a restricted network cluster using the OpenShift Update Service
+
+include::modules/update-service-overview.adoc[leveloffset=+2]
```

Comment:
Content from line 79
https://github.com/openshift/openshift-docs/blob/main/updating/understanding-the-update-service.adoc